README.md

```diff
@@ -77,6 +77,7 @@ See code for all available configurations.
 | Asus TUF FX504GD                  | `<nixos-hardware/asus/fx504gd>`                    |
 | BeagleBoard PocketBeagle          | `<nixos-hardware/beagleboard/pocketbeagle>`        |
 | Dell Latitude 3480                | `<nixos-hardware/dell/latitude/3480>`              |
+| Dell Precision 5530               | `<nixos-hardware/dell/precision/5530>`             |
```

Comment:
Please also add to `flake.nix` apart from that it looks good to me.

Reply:
done, but for some reason github closed the PR when I force pushed...